CONTRIBUTING.md

```diff
@@ -0,0 +1,135 @@
+# Contributing to the SAFE Network
+
+:tada: Thank you for your interest in contributing to the SAFE Network! :tada:
+
+This document is a set of guidelines for contributing to the SAFE Network. These are guidelines, not rules. This guide is designed to make it easy for you to get involved.
+
+As an open source project, we're excited to encourage and accept contributions from our community and beyond, and are striving to make that as easy and enjoyable as possible.
+
+Notice something amiss? Have an idea for a new feature? Feel free to create an Issue in this GitHub repository about anything that you feel could be fixed or improved. Examples include:
+- Bugs, crashes
+- Enhancement ideas
+- Unclear documentation
+- Lack of tutorials and hello world examples
+- ... and more
+
+See our [Issues and Feature Requests](#issues-and-feature-requests) section below for further information on creating new Issues.
+
+Of course, after submitting an Issue you are free to tackle the problem yourself, or pick up any of the other outstanding issues yet to be actioned - see the [Development](#development) section below for more information.
+
+Further below we also have some information on where to go for [further help and support](#support).
```

Comment:
Further below we also have some information on where to go fo -> Further support is available here ?